articles/azure-fluid-relay/quickstarts/quickstart-dice-roll.md

```diff
@@ -27,7 +27,7 @@ You'll also need the following software installed on your computer.
 
 ## Getting Started Locally
 
-First, you'll need to download the sample app from GitHub. Open a new command window and navigate to the folder where you'd like to download the code and use Git to clone the [FluidHelloWorld repo](https://github.com/microsoft/FluidHelloWorld). The cloning process will create a subfolder named FluidHelloWorld with the project files in it.
+First, you'll need to download the sample app from GitHub. Open a new command window and navigate to the folder where you'd like to download the code and use Git to clone the [FluidHelloWorld repo](https://github.com/microsoft/FluidHelloWorld/tree/main-azure). The cloning process will create a subfolder named FluidHelloWorld with the project files in it.
```

Comment:
The folder name created by the clone will change if you clone a different part of the repo. Please update the folder name in the text following the link.

Reply:
the clone instructions are left unchanged, but since the instructions ask you to check out the `main-azure` branch, this updates the link to the repo to be to that branch. I added an extra cause to call this out once more, since I realized if you follow the standard "git clone" process from a non-main branch you'll still check out the default branch when using no other arguments.